spec/rails_helper.rb

```diff
@@ -59,14 +55,14 @@
 
   config.include Devise::TestHelpers, type: :controller
 
-  config.before(:suite) do
-    DatabaseCleaner.strategy = :transaction
-    DatabaseCleaner.clean_with(:truncation)
-  end
+  #config.before(:suite) do
+  #  DatabaseCleaner.strategy = :transaction
```

Comment:
I thought this was just repeating everything, since we have:
https://github.com/fsek/web/blob/master/spec/support/database_cleaner.rb